app.rb

```diff
@@ -0,0 +1,164 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './book'
+require './rental'
+
+class App
+  attr_accessor :book, :people, :rentals
+
+  def initialize
+    @book = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    puts hello
+  end
+
+  def hello
+    puts 'You are welcome to School Library App!'
+    puts ''
+    puts 'Please, enter a number of your choose:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    option = gets.chomp.to_i
+    case option
+    when 1..6 then option_func(option)
+    when 7 then puts 'Thank you for using this app'
+    else puts 'please enter a valid number'
+    end
+  end
```

Comment:
You did a great job with the project. However, there is this issue, when I enter the wrong number which is not in the options, it quits when it should loop back to the menu. I recorded [this video](https://www.loom.com/share/56963a8f90a34f13aad5424761ea2613) where I demonstrated the problem. Kindly fix this issue. This is a part of the project requirements.